src/main/java/org/jenkins/tools/test/model/PluginCompatTesterConfig.java

```diff
@@ -47,211 +41,167 @@
  */
 public class PluginCompatTesterConfig {
 
-    private static final Logger LOGGER = Logger.getLogger(PluginCompatTesterConfig.class.getName());
-
-    // A working directory where the tested plugin's sources will be checked out
-    public File workDirectory;
-
-    // Path for maven settings file where repository will be provided allowing to
-    // download jenkins-core artifact (and dependencies)
-    private File m2SettingsFile;
-
     // The megawar
-    private File war;
+    @NonNull private final File war;
```

Comment:
What looks like needless churn in this file is just sorting these fields, getters, and setters in the same order as they appear in the main CLI Java source file.